src/main/java/frc/robot/CommandFactory.java

```diff
@@ -37,6 +40,65 @@ public CommandFactory(
         this.superStructure = superStructure;
     }
 
+    private boolean atTargetPosition(double currentPosition, double targetPosition, double threshold) {
+        return Math.abs(currentPosition - targetPosition) <= threshold;
+    }
```

Comment:
Instead of making a general method here, you can add a method in the Elevator and Arm subsystems.

`boolean atPosition(double elevatorPosition)`
`boolean atPosition(double shoulderPosition, double wristPosition)`